src/res/apps.yaml

```diff
@@ -601,4 +601,37 @@ items:
     otherlinks:
       - https://cameroun.geo.sm
     image: geosm.jpeg
-#--------------------------------------------------------------
\ No newline at end of file
+#--------------------------------------------------------------
+items:
```

Comment:
```suggestion
```

Reply:
This needs to be removed, I recommend you to try and run the application locally before submitting the PR

Reply:
Ohh ok I see the error. Let me correct that !